### PR TITLE
Remove single

### PR DESCRIPTION
**Pull request by jonahkohn**

### CODE REVIEW

File: crop_mips.ijm [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+    else{
+        load_multiple(chosen_folder);
+    }
+
+} // macro
+
+
+function crop(path, destination, x, y) {
+
+    makeRectangle(x, y, 1024, 1024);
```
Comment (berl) [inline crop_mips.ijm:18]: Do the (x,y) here correspond to the center of the rectangle or ?

**Review by berl — APPROVED** [review]
Refactoring the processing into a class is a huge improvement and eliminating the duplicate .py files for single vs multiple really makes it more readable. Nice work

**Comment by berl** [comment]
One other comment:  the documentation stuff should probably have been merged in first before starting on the rest of the refactoring. Next time, I'd suggest making a new branch

File: crop_mips.ijm (hunk 1 above)
Comment (jonahkohn) [reply crop_mips.ijm:18]: x, y correspond to the top left. (0, 0) is the top left of the image, and X, Y move positively right and down from there.

**Pull request merged**